packages/webapp-alternative/webpack.prod.js

```diff
@@ -0,0 +1,11 @@
+const { merge } = require('webpack-merge');
+const common = require('./webpack.common.js');
+const TerserPlugin = require("terser-webpack-plugin");
+
+module.exports = merge(common, {
+  mode: 'production',
+  optimization: {
+    minimize: true,
+    minimizer: [new TerserPlugin()],
```

Comment:
Seems like we can omit this configuration with Terser because it is used by default according to the documentation. https://webpack.js.org/guides/production/#minification